src/getFilenameValidationData.test.ts

```diff
@@ -0,0 +1,287 @@
+/* tslint:disable:mocha-no-side-effect-code mocha-avoid-only */
```

Comment:
I haven't encountered issues with TSLint in Jest tests using our shared TSLint configuration. Are those needed?

Reply:
No, I only needed them during development. Will revert.